js/services.js

```diff
@@ -228,6 +229,21 @@ angular.module('wpApp.services', [])
 			});
 	}
 	
+	function getNextID() {
+		return $q.when( _db.allDocs( { descending: true, limit: 1 } ) )
```

Comment:
Let me check some things. When doing it that way, I may have been referencing a doc from before PouchDB supported promises.